tools/lkl/lib/virtio_net_vde.c

```diff
@@ -1,6 +1,6 @@
+#include <stdio.h>
 #ifdef CONFIG_AUTO_LKL_VIRTIO_NET_VDE
```

Comment:
Need to change this to CONFIG_AUTO_POSIX_LKL.

Reply:
I think CONFIG_AUTO_LKL_VIRTIO_NET_VDE should be okay as your commit (https://github.com/tavip/linux/commit/20ed06560794f99fdcb6beaf0c764d60f1ecd0b5) does, no ?

Reply:
you are right :)

Reply:
okay